mmpose/models/heads/heatmap_heads/simcc_head.py

```diff
@@ -275,18 +289,17 @@ def loss(
         )
 
         target_coords = np.concatenate(
-            [d.gt_instances.keypoints for d in batch_data_samples], axis=0)
+            [d.gt_instances.transformed_keypoints for d in batch_data_samples],
```

Comment:
Why do we use `transformed_keypoints` instead of `keypoints` here? In the training pipeline `transformed_keypoints` will be packed as `keypoints` so they should be identical. Actually I think `transformed_keypoints` serves as an intermediate variable in pipelines and shouldn't be packed and used by models.